udata_front/theme/gouvfr/assets/js/components/dataset/card-lg.vue

```diff
@@ -163,13 +163,10 @@
 import { defineComponent, PropType } from "vue";
 import { useI18n } from "vue-i18n";
 import useLicense from "../../composables/useLicense";
-import { OrganizationNameWithCertificate, useOwnerName, formatRelativeIfRecentDate} from "@etalab/data.gouv.fr-components";
+import { OrganizationNameWithCertificate, useOwnerName, formatRelativeIfRecentDate, QualityItem, QualityScore, Toggletip } from "@etalab/data.gouv.fr-components";
```

Comment:
Can't we use `QualityComponent.vue` directly in dataset card?

Reply:
Exactly ! It will be done in my next PR dedicated to the dataset cards